src/main/kotlin/io/kotest/extensions/wiremock/MatchJsonSchemaPattern.kt

```diff
@@ -0,0 +1,23 @@
+@file:Suppress("RedundantVisibilityModifier")
+
+package io.kotest.extensions.wiremock
+
+import com.github.tomakehurst.wiremock.matching.MatchResult
+import com.github.tomakehurst.wiremock.matching.StringValuePattern
+import io.kotest.assertions.json.schema.JsonSchema
+import io.kotest.assertions.json.schema.shouldMatchSchema
+import wiremock.com.fasterxml.jackson.annotation.JsonProperty
+
+public class MatchJsonSchemaPattern(
+   @JsonProperty("schema") private val schema: JsonSchema
+) : StringValuePattern("json schema") {
```

Comment:
This string should be the serial form of the Schema, when/if we have support for serializing schemas back to JSON..